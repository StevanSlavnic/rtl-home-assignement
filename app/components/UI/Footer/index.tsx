import React from "react";

export default function Footer() {
  return (
    <footer
      data-testid="footer"
      className="bg-gray-800 text-white text-center p-4"
    >
      Footer
    </footer>
  );
}
