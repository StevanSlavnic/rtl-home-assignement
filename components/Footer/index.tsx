import React from "react";

export default function Footer() {
  return (
    <footer
      data-testid="footer"
      className="h-[300px] bg-gray-800 text-white text-center p-4"
    ></footer>
  );
}
