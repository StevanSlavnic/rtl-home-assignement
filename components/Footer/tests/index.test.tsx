import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Footer from "../index";

describe("Footer", () => {
  it("renders the Footer component", () => {
    render(<Footer />);

    const footerElement = screen.getByTestId("footer");
    expect(footerElement).toBeInTheDocument();
    expect(footerElement).toMatchSnapshot();
  });
});
