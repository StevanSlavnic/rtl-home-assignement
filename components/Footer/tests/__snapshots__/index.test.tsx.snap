// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the Footer component 1`] = `
<footer
  class="h-[300px] bg-gray-800 text-white text-center p-4"
  data-testid="footer"
/>
`;
